frontend/src/views/users.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import messages from './messages';
+import { FormattedMessage } from 'react-intl';
+import { SearchNav, UsersTable } from '../components/userList';
+
+export const UsersList = () => {
+  const [filters, setFilters] = useState({ level: 'ALL', role: 'ALL', username: '', page: 1 });
+
+  return (
+    <div>
+      <h3 className="barlow-condensed f2 ma0 pv3 dib v-mid ttu pl2 pl0-l">
+        <FormattedMessage {...messages.userList} />
+      </h3>
+      <div className="w-80 bg-white cf pa3 mb4">
+        <SearchNav filters={filters} setFilters={setFilters} />
```

Comment:
Let's mode the SearchNav to above the users list to follow the same interface pattern:

![image](https://user-images.githubusercontent.com/666291/74843141-445dd580-530a-11ea-81a6-fb0809e3b5f2.png)

![image](https://user-images.githubusercontent.com/666291/74843235-648d9480-530a-11ea-838c-05d91da6e97b.png)

![image](https://user-images.githubusercontent.com/666291/74843279-73744700-530a-11ea-9fbc-92d9a6ee717a.png)